### DISCUSSION TITLE
Is there any convenient way to add a value when the dictionary key does not exist?

### DISCUSSION THREAD

**Discussion by zms9110750** [dotnet/csharplang#6032] [Q&A] [answered]
My current code:
```
GetValue=>
dic.TryGetValue(name, out T com) ? com : dic[name] = Find( name);
```
Lazy can tell him how to get the value in advance. Is there any way to use it with a dictionary?

**Comment by huoyaoyuan** [comment, marked as answer]
This is not a language specific question, and should belong to the dotnet/runtime repo.
See also https://github.com/dotnet/runtime/discussions/67376.
